spa/src/app.tsx

```diff
@@ -17,7 +17,8 @@ initSentry();
 const SentryRoutes = Sentry.withSentryReactRouterV6Routing(Routes);
 
 const App = () => {
-
+	// to reset the local storage value on app reload
+	localStorage.removeItem("isPopupBlocked");
```

Comment:
Please add a check first before removing it. 
```suggestion
	if (localStorage.getItem("isPopupBlocked") === null) {
	  // to reset the local storage value on app reload
	  localStorage.removeItem("isPopupBlocked");
}
```